iFixit/pulldasher
# Notification: Open pull on click

This adds a new feature to the Pulldasher notifications: they now open the pull they're about when you click them. This is much more convenient, since the majority of the time you want to open the pull if you're interacting with the notification.

## QA Notes
1. Run `npm run frontend:start`
2. Click one of the notifications which appears and confirm that the pull is opened in a new tab.
